example/feature_demo/demo_multi_file.pb.go

```diff
@@ -1,12 +1,13 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.27.1
-// 	protoc        v3.15.7
+// 	protoc-gen-go v1.23.0
+// 	protoc        v3.17.1
 // source: feature_demo/demo_multi_file.proto
 
 package example
 
 import (
+	proto "github.com/golang/protobuf/proto"
```

Comment:
I don't think this is right, because it uses deprecated go bindings for protocol buffers, instead we should use  google.golang.org/protobuf and also I see version downgrade from protoc-gen-go v1.27.1 to v1.23.0